app/src/main/java/com/release/keyneez/presentation/main/search/SearchActivity.kt

```diff
@@ -0,0 +1,47 @@
+package com.release.keyneez.presentation.main.search
+
+import android.os.Bundle
+import android.view.KeyEvent
+import androidx.activity.viewModels
+import com.release.keyneez.R
+import com.release.keyneez.databinding.ActivitySearchBinding
+import com.release.keyneez.util.binding.BindingActivity
+import com.release.keyneez.util.extension.hideKeyboard
+import com.release.keyneez.util.extension.setOnSingleClickListener
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class SearchActivity : BindingActivity<ActivitySearchBinding>(R.layout.activity_search) {
+    lateinit var searchAdapter: SearchAdapter
```

Comment:
```suggestion
    private lateinit var searchAdapter: SearchAdapter
```
외부에서 사용하지 않는 멤버 필드는 `private`을 지정해주세요!